mne/utils/config.py

```diff
@@ -719,7 +739,16 @@ def sys_info(fid=None, show_paths=False, *, dependencies="user", unicode=True):
         except Exception:
             unavailable.append(mod_name)
         else:
-            out(f"{pre}☑ " if unicode else " + ")
+            mark = "☑" if unicode else "+"
+            mne_extra = ""
+            if mod_name == "mne" and check_version:
+                timeout = 2.0 if check_version is True else float(check_version)
+                mne_version_good, mne_extra = _check_mne_version(timeout)
+                if mne_version_good is None:
+                    mne_version_good = True
+                elif not mne_version_good:
+                    mark = "☒" if unicode else "X"
+            out(f"{pre}{mark} " if unicode else " {mark} ")
```

Comment:
```suggestion
            out(f"{pre}{mark} " if unicode else f" {mark} ")
```

I think you need the `f` in both places, no?